cypress/integration/ProcessorTest.ts

```diff
@@ -237,41 +261,43 @@ describe.skip("Processor Test", () => {
       );
     });
 
-    it("Add filter row", () => {
-      filters[1] = ["data.surname", "String equals", "White"];
-
-      cy.ouiaId("add-filter", "PF4/Button").click();
-
-      cy.ouiaId("item-1").within(() => {
-        cy.ouiaId("filter-key", "PF4/TextInput").type(filters[1][0]);
-        cy.ouiaId("filter-type", "PF4/FormSelect").select(filters[1][1]);
-      });
-      //The filter-value was detached from DOM and we need to find the context again.
-      cy.ouiaId("item-1")
-        .ouiaId("filter-value", "PF4/TextInput")
-        .type(filters[1][2]);
+    onlyOn(bugPresent, () => {
```

Comment:
At the place of 'bugPresent' should be a condition. Empty string is false and not empty string is true. I have found this method in constractions like skip if  values in configuration is present or equal, skip if screenresolution is lower than, etc. I have choosen this method to be clear why it is skipped. But I understand that it is not ideal. It could be useful to create something like KIG in cypress. I hope that i delete this method in few weeks.

Reply:
ok, thank you, approving